### ISSUE TITLE
Protect root group

### ISSUE THREAD

**Issue by vaygr** [firecat53/keepmenu#126]
Thanks for this nice little piece of software!

I've been playing around with keepmenu lately trying to break it, and noticed an issue. Not sure if it's by design.

It seems it's easy to wipe out the root group in keepmenu. The database then becomes effectively useless: I can't add the root group back because adding any group requires specifying parent group, which doesn't exist in the first place. When I then try to open the database in KeePassXC I get an error `Error while reading the database: No root group`.

In KeePassXC the root group is protected, so maybe it makes sense to do the same in keepmenu?

**Comment by firecat53** [comment]
Yep, I can reproduce. Makes sense to not allow the root group to be deleted.
